docs/template_plugin/tests/functional/op_reference/extract_image_patches.cpp

```diff
@@ -0,0 +1,248 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include "openvino/opsets/opset3.hpp"
+#include "openvino/opsets/opset1.hpp"
+#include "base_reference_test.hpp"
+
+using namespace reference_tests;
+using namespace ov;
+
+namespace {
+struct ExtractImagePatchesParams {
+    Tensor data;
+    Shape sizes;
+    Strides strides;
+    Shape rates;
+    op::PadType autoPad;
+    Tensor expectedResult;
+    std::string testcaseName;
+};
+
+struct Builder : ParamsBuilder<ExtractImagePatchesParams> {
+    REFERENCE_TESTS_ADD_SET_PARAM(Builder, data);
+    REFERENCE_TESTS_ADD_SET_PARAM(Builder, sizes);
+    REFERENCE_TESTS_ADD_SET_PARAM(Builder, strides);
+    REFERENCE_TESTS_ADD_SET_PARAM(Builder, rates);
+    REFERENCE_TESTS_ADD_SET_PARAM(Builder, autoPad);
+    REFERENCE_TESTS_ADD_SET_PARAM(Builder, expectedResult);
+    REFERENCE_TESTS_ADD_SET_PARAM(Builder, testcaseName);
+};
+
+class ReferenceExtractImagePatchesTest : public testing::TestWithParam<ExtractImagePatchesParams>, public CommonReferenceTest {
+public:
+    void SetUp() override {
+        auto params = GetParam();
+        function = CreateFunction(params);
+        inputData = {params.data.data};
+        refOutData = {params.expectedResult.data};
+    }
+
+    static std::string getTestCaseName(const testing::TestParamInfo<ExtractImagePatchesParams>& obj) {
+        auto param = obj.param;
+        std::ostringstream result;
+        result << "dType=" << param.data.type;
+        result << "_dShape=" << param.data.shape;
+        result << "_sizes=" << param.sizes;
+        result << "_strides=" << param.strides;
+        result << "_rates=" << param.rates;
+        result << "_autoPad=" << param.autoPad;
+        result << "_eType=" << param.expectedResult.type;
+        if (param.testcaseName != "") {
```

Comment:
How about this?

        result << "_eShape=" << param.expectedResult.shape;
        if (!param.testcaseName.empty()) {
            result << "_=" << param.testcaseName;
        }

Reply:
Updated, thanks for your comment